### PR TITLE
🐛 Replace references to ks-0.20 with main

### PR DESCRIPTION
**Pull request by MikeSpreitzer**
## Summary
This PR replaces references to branch `ks-0.20` with references to branch `main` or relative references.

This is part of the merging of branch `ks-0.20` into `main`.

## Related issue(s)

Fixes #

### CODE REVIEW

File: test/e2e/multi-cluster-deployment/README.md [hunk 1]
```diff
@@ -14,11 +14,10 @@ cd test/e2e/multi-cluster-deployment
 
 ## Running the test steps manually
 ### Setup Kubestellar
-- Clone the repo and checkout the ks-0.20 branch
+- Clone the repo and cd to it. Naturally, if the branch that you are reading is not `main` then use that branch.
```
Comment (ezrasilvera) [inline test/e2e/multi-cluster-deployment/README.md:17]: not clear to me what is "the branch that you are reading "  , you mean the branch the user want to test ?

Comment (MikeSpreitzer) [reply test/e2e/multi-cluster-deployment/README.md:17]: Someone reading this content is looking at a particular branch. Most likely on the web, e.g. at https://github.com/kubestellar/kubestellar/tree/main/test/e2e/multi-cluster-deployment . See, there is a branch name in there.

File: test/e2e/multi-cluster-deployment/run-test.sh [hunk 2]
```diff
@@ -14,7 +14,7 @@
 # limitations under the License.
 
 # This is an end to end test of multi cluster deployement.  
-# For readable instructions, please visit https://github.com/kubestellar/kubestellar/tree/ks-0.20/docs/content/v0.20
+# For readable instructions, please visit ../../../docs/content/v0.20
```
Comment (ezrasilvera) [inline test/e2e/multi-cluster-deployment/run-test.sh:17]: So what exactly is the definition and role of the  README under the test directory ?

File: test/e2e/singleton-status/run-test.sh [hunk 3]
```diff
@@ -14,7 +14,7 @@
 # limitations under the License.
 
 # This is an end to end test of multi cluster deployement.  
-# For readable instructions, please visit https://github.com/kubestellar/kubestellar/tree/ks-0.20/docs/content/v0.20
+# For readable instructions, please visit ../../../docs/content/v0.20
```
Comment (ezrasilvera) [inline test/e2e/singleton-status/run-test.sh:17]: Same

File: test/e2e/multi-cluster-deployment/run-test.sh (hunk 2 above)
Comment (MikeSpreitzer) [reply test/e2e/multi-cluster-deployment/run-test.sh:17]: Most importantly: not changed in this PR.
FYI, it is about how to run the tests.

Comment (ezrasilvera) [reply test/e2e/multi-cluster-deployment/run-test.sh:17]: I accept the "not changed"  But I prefer to point to the test README as the source of truth (will ad @pdettori link as an additional optional source of info).

**Pull request merged**